dist/licenses.txt

```diff
@@ -12,15 +12,6 @@ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLI
 
 @actions/github
 MIT
-The MIT License (MIT)
```

Comment:
what happened to the license here?

Reply:
Not really sure what's went wrong with the license. I just ran the `ncc build` command  to generate the license. I guess the tool has done some re-ordering of licenses which is why it appears like this.